setup.py

```diff
@@ -10,7 +10,7 @@
     author='Andy McKay',
     author_email='andym@mozilla.com',
     license='BSD',
-    install_requires=['statsd==2.1.2'],
+    install_requires=['statsd>=2.1.2'],
```

Comment:
you can use `install_requires = ['statsd >= 2.1.2, < 3.0.0']` if you trust statsd to keep to semver